### DISCUSSION TITLE
HAA for Sonoff Zigbee hub

### DISCUSSION THREAD

**Discussion by Stetocasa** [RavenSystem/esp-homekit-devices#1742] [New Features]
Guys, I’m going to flash my Sonoff zigbee hub to Tasmota looking the multiples guide on the web.
100% of guide is for HA usage, my goal will be continue the process for HAA

my biggest concern is, how pair child devices?
Theoretically HAA will show an hub accessory (not an outlet). Do you know how manage the addition on zigbee Sonoff devices?

thanks

**Comment by i3laze** [comment]
You mention both Tasmota and HAA, it’s confusing.

Anyway currently HAA doesn’t support Zigbee or hubs.
All your memorized ZigBee relays will be definitely wiped from memory.
You should backup (_read_flash_) your stock firmware before trying HAA.

**Comment by RavenSystem** [comment]
Please, use Discussions section and leave Issues section only for issues. Thanks!

https://github.com/RavenSystem/esp-homekit-devices/wiki#issues
